x_transformers/attend.py

```diff
@@ -370,7 +370,7 @@ def flash_attn(
         # convert from bool to float
 
         if exists(attn_bias):
-            attn_bias = rearrange(attn_bias, 'h i j -> 1 h i j').expand(batch, heads, -1, -1)
+            attn_bias = attn_bias.expand(batch, heads, -1, -1)
```

Comment:
cool, but does this still work if the bias has 3 dimensions?

Reply:
yes, expand will broadcast along the last dimensions. I can add a more verbose rearrange back if you prefer that

Reply:
ah nice, didn't know that about expand! til